packages/frontend/src/providers/__mocks__/ReactQueryProvider.tsx

```diff
@@ -0,0 +1,22 @@
+import { QueryClientProvider } from '@tanstack/react-query';
+import { FC, PropsWithChildren } from 'react';
+import { createQueryClient } from '../ReactQueryProvider';
+
+const ReactQueryProvider: FC<PropsWithChildren> = ({ children }) => {
+    const queryClient = createQueryClient({
+        queries: {
+            retry: false,
+        },
+        mutations: {
+            retry: false,
+        },
```

Comment:
overrides necessary for testing https://tanstack.com/query/v4/docs/framework/react/guides/testing#turn-off-retries